superset/assets/src/SqlLab/actions/sqlLab.js

```diff
@@ -111,11 +110,9 @@ export function fetchQueryResults(query) {
 
     return SupersetClient.get({
       endpoint: `/superset/results/${query.resultsKey}/`,
-      parseMethod: 'text',
     })
-      .then(({ text = '{}' }) => {
-        const bigIntJson = JSONbig.parse(text);
-        dispatch(querySuccess(query, bigIntJson));
+      .then(({ json = {} }) => {
```

Comment:
could simplify this to `.then(({ json })) => dispatch(querySuccess(query, json))` which would also return a promise (this is ideal for chained promises esp if it's a dispatch because you can continue to chain calls)